experimental/torch_xla2/torch_xla2/ops/jaten.py

```diff
@@ -1989,3 +1990,48 @@ def _aten_native_batch_norm(input, weight, bias, running_mean, running_var, trai
     return torch.ops.aten._native_batch_norm_legit(input, weight, bias, running_mean, running_var, training, momentum, eps)
   else:
     return torch.ops.aten._native_batch_norm_legit_no_training(input, weight, bias, running_mean, running_var, momentum, eps)
+
+
+@op(torch.ops.aten.normal, needs_env=True)
```

Comment:
Do you know when should we use `needs_env` for those ops?

Reply:
if the ops also want to have an keyword argument `env` passed to it, it should ask with needs_env. Usually random numbers generators needs it